packages/commerce-sdk-react/src/hooks/ShopperBaskets/query.ts

```diff
@@ -50,6 +51,7 @@ export const useBasket = (
 
     // For some reason, if we don't explicitly set these generic parameters, the inferred type for
     // `Data` sometimes, but not always, includes `Response`, which is incorrect. I don't know why.
+    // @ts-ignore Fix
```

Comment:
The new react-query v5 had a MAJOR overhaul of its type definitions, including removing of tons of overloads.

Now, only one function parameter type is available, which is a super good news for us, because in the past, we've had issues to create types that satisfy all of the overloads. This was a major blocker for us creating a DRY implementation, and we had to duplicate the implementation for every Shopper api.

I'm going to create a new ticket to re-implement the useQuery and useMutation typings and hopefully we can use generics to DRY the implementation.

This would also drastically decrease the bundle size.

Ticket to be created.